PSKoans/Koans/Constructs and Patterns/AboutEnumerations.Koans.ps1

```diff
@@ -0,0 +1,584 @@
+                to file system rights.
+            #>
+
+            It 'represents each flag value with a single bit' {
+                <#
+                    A name in the enumeration normally represents a single bit in a numeric value.
+
+                    PowerShell enumerations use Int32 by default. PowerShell Core can use other integer types as
+                    demonstrated in the previous context.
+
+                    32 unique flags can be expressed in an enumeration based on Int32. Each value will be double that
+                    of the last. Therefore expected values are 1, 2, 4, 8, 16, and so on. These are the values of
+                    individual bits.
+
+                    The [Flags()] attribute is placed above the enum keyword.
+
+                    The example below is based on 4 different flags.
+                #>
+
+                $script = {
+                    [Flags()]
+                    enum Bit {
+                        None = 0
+                        Bit1 = 1
+                        Bit2 = 2
+                        Bit3 = 4
+                        Bit4 = 8
+                    }
+
+                    $using:Value -as [Bit] -as [String]
+                }
+
+                $Value = 2
+                $Name = Start-Job -ScriptBlock $script | Receive-Job -Wait
+
+                '____' | Should -Be $Name
+
+                # When more than one flag is described, the value returned is a comma separated list.
+
+                $Value = 2 -bor 8 # 10
+                $Name = Start-Job -ScriptBlock $script | Receive-Job -Wait
+
+                '____' | Should -Be $Name
+            }
+
+            It 'can represent multiple flag names with a single value' {
+                # A list of names can be converted to a single flag value.
+
+                $script = {
+                    [Flags()]
+                    enum Bit {
+                        None = 0
+                        Bit1 = 1
+                        Bit2 = 2
+                        Bit3 = 4
+                        Bit4 = 8
+                    }
+
+                    $using:Names -as [Bit] -as [Int32]
+                }
+
+                <#
+                    The names are a single string with commas separating values, not an array.
+                    Spaces are discarded when converting the value.
+                #>
+
+                $Names = '____, ____'
+                $Value = Start-Job -ScriptBlock $script | Receive-Job -Wait
+
+                __ | Should -Be $Value
+            }
+
+            It 'allows you to test for the presence of a given flag' {
+                <#
+                    A value which has multiple flags set is provided as a single value.
+
+                    The presence of individual flag can be tested in either of two ways.
+
+                    The HasFlag may be called on an enumeration value:
```

Comment:
```suggestion
                    One way is to call the HasFlag() method, which may be called on an enumeration value:
```

This could also be the same paragraph as the sentence immediately above, I think?